ci/setup-build-test-gdk.ps1

```diff
@@ -8,7 +8,8 @@ param(
   [string] $test_repo_branch = "master",
   [string] $test_repo_url = "https://github.com/spatialos/UnrealGDKTestGyms.git",
   [string] $test_repo_relative_uproject_path = "Game\GDKTestGyms.uproject",
-  [string] $test_repo_map = "EmptyGym"
+  [string] $test_repo_map = "EmptyGym",
+  [string] $spatial_cli_version = "20191106.121025.bda19848a2"
```

Comment:
Our CI was previously using an unpinned version. This is a fix to improve reproduciblity.

Reply:
Spatial is baked into the agent right? If so, why are we changing the version of it during a build?